GiniVision/Classes/ImageAnalysisNoResultsContainerViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ImageAnalysisNoResultsContainerViewController.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 10/16/17.
+//
+
+import Foundation
+
+internal class ImageAnalysisNoResultsContainerViewController: UIViewController, ContainerViewController {
+    
+    // Container attributes
+    internal var containerView     = UIView()
+    internal var contentController = UIViewController()
+    
+    // Resources
+    fileprivate let closeButtonResources = PreferredButtonResource(image: "navigationCameraClose", title: "ginivision.navigationbar.camera.close", comment: "Button title in the navigation bar for the close button on the camera screen", configEntry: GiniConfiguration.sharedConfiguration.navigationBarCameraTitleCloseButton)
+    fileprivate let backButtonResources = PreferredButtonResource(image: "arrowBack", title: "ginivision.navigationbar.analysis.back", comment: "Button title in the navigation bar for the back button on the analysis screen", configEntry: GiniConfiguration.sharedConfiguration.navigationBarAnalysisTitleBackButton)
+
+    init(canGoBack: Bool = true) {
+        super.init(nibName: nil, bundle: nil)
+        
+        // Configure content controller
+        let imageAnalysisNoResultsViewController: ImageAnalysisNoResultsViewController
+        
+        if canGoBack {
+            imageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController()
+        } else {
+            imageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController(bottomButtonText: nil, bottomButtonIcon: nil)
+        }
+        
+        imageAnalysisNoResultsViewController.didTapBottomButton = { [weak self] in
+            self?.backToCamera()
+        }
+        contentController = imageAnalysisNoResultsViewController
+        
+        // Configure colors
+        view.backgroundColor = GiniConfiguration.sharedConfiguration.backgroundColor
+        
+        // Configure close button
+        if canGoBack {
+            setupLeftNavigationItem(usingResources: backButtonResources, selector: #selector(backToCamera))
+        } else {
+            setupLeftNavigationItem(usingResources: closeButtonResources, selector: #selector(close))
+        }
+        
+        // Configure view hierachy
```

Comment:
Useless and also not true. You are not configuring the hierarchy. You are just adding a subview.